cms/static/js/views/course_info_handout.js

```diff
@@ -50,10 +50,14 @@ define(['js/views/baseview', 'codemirror', 'common/js/components/views/feedback_
             },
 
             onSave: function(event) {
+                var dataToBeSaved = this.$codeMirror.getValue();
                 $('#handout_error').removeClass('is-shown');
                 $('.save-button').removeClass('is-disabled').attr('aria-disabled', false);
                 if ($('.CodeMirror-lines').find('.cm-error').length == 0) {
-                    this.model.set('data', this.$codeMirror.getValue());
+                    if (dataToBeSaved === '') {
```

Comment:
@cahrens the issue (referred to by the ticket) occurred only if the dataToBeSaved is left blank. i.e the implementation saves the content if it contains one or more blank spaces. However it rejects to save if the content is left completely blank. I had the thought in mind that anything written in the content should be saved as it is without any validation of what's written within that content. (even if it contains spaces or any other special characters). Let me know if this implementation needs to change.